### DISCUSSION TITLE
<< Implementation of implicit solvation >>

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#801] [Q&A]
Hi,

From xTB documentation:
"..._In this chapter, all neccessary information will be given in order to use the implicit solvent model ALPB in xTB calculations..."

It is not clear to me in which xTB calculations the implicit solvent can be used.

It will work for:
1- single point energy?
2- geometry optimization?
3- aISS (docking module)?
4- Molecular dynamics?

Regards,

Camps

**Comment by haneug** [comment]
For energies and nuclear gradients, you can use implicit solvation with GFN1-xTB, GFN2-xTB, and GFN-FF. So you can employ it for 1. single point energy, 2. geometry optimization, and 4. molecular dynamics. 
I am not that familiar with the docking module and I am not sure what effect implicit solvation can have for the xTB-IFF that is employed for the docking, but you can definitely use implicit solvation for the geometry optimizations of the final ensemble.

**Reply by haneug** [reply]
Maybe @cplett can comment on the docking module?
